portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase

```diff
@@ -0,0 +1,119 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		task ("Given user with admin privileges and navigate to home page") {
+			User.firstLoginPG();
+
+			Navigator.openURL();
+		}
+
+		task ("Adding a page and add blog widget to public layout") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Blogs Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "Blogs Page",
+				widgetName = "Blogs");
+		}
+
+		task ("Going to the page and adding a blog entry") {
+			Navigator.gotoPage(pageName = "Blogs Page");
+
+			BlogsNavigator.openToAddEntry(
+				layoutName = "Blogs Page",
+				siteURLKey = "guest");
+		}
+
+		task ("Creating and publishing a blog") {
+			BlogsEntry.addTitle(entryTitle = "Blogs Entry Title");
+
+			BlogsEntry.addContent(entryContent = "Blogs Entry Content");
+
+			PortletEntry.publish();
+		}
+
+		task ("Go to the blogs page") {
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Guest");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+
+		BlogsEntry.tearDownCP();
+
+		PagesAdmin.openPagesAdmin(siteURLKey = "Guest");
+
+		PagesAdmin.deletePage(pageName = "Blogs Page");
+	}
+
+	@description = "Can Create a Blog"
+	@priority = 5
+	test CanCreateBlog {
+		task ("Given a new additional entry on the page created blog page") {
+			BlogsNavigator.openToAddEntry(
+				layoutName = "Blogs Page",
+				siteURLKey = "guest");
+		}
+
+		task ("And a blog is created") {
+			BlogsEntry.addTitle(entryTitle = "Blogs Entry Title");
+
+			BlogsEntry.addContent(entryContent = "Blogs Entry Content");
+		}
+
+		task ("Published blog") {
+			PortletEntry.publish();
+		}
+
+		task ("Validating the blog title and content") {
+			Navigator.gotoSitePage(
+				pageName = "Blogs Page",
+				siteName = "Guest");
```

Comment:
Hey @crislucas1, if you put these steps in `setUp`, then you can remove this part from your first test since the steps to create the blog and page are already being done by `setUp`